File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/SimpleMecanumBot.java
package org.firstinspires.ftc.teamcode.opmodes;

import org.firstinspires.ftc.teamcode.features.NativeMecanumDrivetrain;
import org.firstinspires.ftc.teamcode.internals.registration.OperationMode;
import org.firstinspires.ftc.teamcode.internals.registration.TeleOperation;

public class SimpleMecanumBot extends OperationMode implements TeleOperation {
    @Override
    public void construct() {
        registerFeature(new NativeMecanumDrivetrain(true));
    }

    @Override
    public void run() {

    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/SimpleMecanumBot.kt
package org.firstinspires.ftc.teamcode.opmodes

import org.firstinspires.ftc.teamcode.features.NativeMecanumDrivetrain
import org.firstinspires.ftc.teamcode.internals.registration.OperationMode
import org.firstinspires.ftc.teamcode.internals.registration.TeleOperation

/**
 * This tests the native mecanum drivetrain.
 *
 *
 * Features: [org.firstinspires.ftc.teamcode.features.NativeMecanumDrivetrain]
 */
class SimpleMecanumBot : OperationMode(), TeleOperation {
    override fun construct() {
        registerFeature(NativeMecanumDrivetrain(false))
    }

    override fun run() {
    }
}
